moskito-backend/src/Entity/Campsite.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CampsiteRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=CampsiteRepository::class)
+ */
+class Campsite
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $street;
+
+    /**
+     * @ORM\Column(type="string", length=5)
+     */
+    private $postalCode;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $place;
+
+    /**
+     * @ORM\Column(type="string", length=30, nullable=true)
+     */
+    private $telephone;
+
+    /**
+     * @ORM\Column(type="string", length=40, nullable=true)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="array")
+     */
+    private $coordinates = [];
```

Comment:
woudnt it be better to have an attribute latitude and an attribute longitude

Reply:
yes, we talked about that in our little mapping-group. I'll let the others know!